extension-jaxrs/src/main/java/io/smallrye/openapi/jaxrs/JaxRsParameterProcessor.java

```diff
@@ -32,21 +32,21 @@
 import io.smallrye.openapi.runtime.util.TypeUtil;
 
 /**
- *
- * Note, javax.ws.rs.PathParam PathParam targets of
- * javax.ws.rs.core.PathSegment PathSegment are not currently supported.
+ * Note, javax.ws.rs.PathParam PathParam targets of javax.ws.rs.core.PathSegment PathSegment are not currently supported.
  *
  * @author Michael Edgar {@literal <michael@xlate.io>}
- *
  */
 public class JaxRsParameterProcessor extends AbstractParameterProcessor {
 
+    private AnnotationScannerContext scannerContext;
+
```

Comment:
This shouldn't be needed. The `scannerContext` in `AbstractParameterProcessor` should be visible.